### ISSUE TITLE
Cinnamon title bars

### ISSUE THREAD

**Issue by Shaddwo** [vinceliuice/WhiteSur-gtk-theme#162]
I am using Manjaro with the Cinnamon desktop, and the title bars in the dark theme have a white accent on the top.

Is this intended?

I have found the same problem with the Materia theme as well

![image](https://user-images.githubusercontent.com/72974062/112852247-627b3500-90c9-11eb-9258-1b806ce81b54.png)

Please fix, great theme by the way

**Comment by Shaddwo** [comment]
Also, the title bars on some apps are slightly off from the right edge of the screen.
![image](https://user-images.githubusercontent.com/72974062/112853006-14b2fc80-90ca-11eb-958d-a244345b15b4.png)

**Comment by Jazzhoo** [comment]
Hi, I have seen that in Cinnamon. I believe these are two different CSDs (maybe one is GTK2 and the other 3 - I don't know).

To reproduce, set different theme in "Controls" and different in "Desktop", look at screenshot (screenshot from internet, I'm on different computer now).
Good app example to reproduce is Manjaro Cinnamon's zip app (i know there were some other but the zipping app is the one i remember now)
![cinnamon-appearance](https://user-images.githubusercontent.com/53535260/114264755-a4b74700-99e4-11eb-86bb-274c62890c2b.png)

In other words, some applications decorations are styled based on what you set in "Controls" and some based what you set in "Desktop". Apparently, even If you set up WhiteSur in both, the style of these variants is slightly different, or the application itself forces alternative layout.

If that's intended or not, I don't know.

Cheers,
J.
